src/internal/observable/pairs.ts

```diff
@@ -8,7 +8,7 @@ import { Subscription } from '../Subscription';
  *
  * <span class="informal">Turn entries of an object into a stream.</span>
  *
- * <img src="./img/pairs.png" width="100%">
+ * <img src="/assets/images/marble-diagrams/pairs.png" width="100%">
```

Comment:
This should be changed to just 'pairs', similar to like you did for `tap`

Reply:
done